tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/AwsSqsCommon.cs

```diff
@@ -56,5 +59,28 @@ public static Scope CreateScope(Tracer tracer, string operation, out AwsSqsTags
             // or we couldn't populate it completely (some tags is better than no tags)
             return scope;
         }
+
+        public static string GetQueueName(string queueUrl)
+        {
+            if (queueUrl is null)
+            {
+                return queueUrl;
+            }
+
+            return queueUrl.Split('/').Last();
+        }
+
+        internal static string GetOperationName(Tracer tracer, string spanKind) => tracer.CurrentTraceSettings.Schema.Version switch
+        {
+            SchemaVersion.V0 => SqsRequestOperationName,
+            _ => GetOperationNameHelper(spanKind)
+        };
+
+        internal static string GetOperationNameHelper(string spanKind) => spanKind switch
+        {
+            SpanKinds.Consumer => $"{SnsOperationName}.process",
+            SpanKinds.Producer => $"{SnsOperationName}.send",
+            _ => $"{SnsOperationName}.request"
+        };
     }
```

Comment:
These methods need unit testing, where would this test live?

Note: These are already covered by integration testing.

Reply:
I believe you could put them in `Datadog.Trace.ClrProfiler.Managed.Tests.AutoInstrumentation.AWS` project and namespace